main.ts

```diff
@@ -404,6 +414,37 @@ class Translator {
         this.emit('}');
         break;
 
+      case ts.SyntaxKind.ImportDeclaration:
+        var importDecl = <ts.ImportDeclaration>node;
+        this.emit('import');
+
+        this.visitExternalModuleReferenceExpr(importDecl.moduleSpecifier);
+
+        if (importDecl.importClause) {
+          this.emit('show');
+          this.visit(importDecl.importClause);
+        }
```

Comment:
done. Better to start restrictive and fail fast, so we discover which of our assumptions are invalid.